web/src/integration_url_modal.ts

```diff
@@ -57,6 +64,68 @@ export function show_generate_integration_url_modal(api_key: string): void {
             );
         });
 
+        function render_config(config: ConfigOption[]): void {
+            const $configContainer = $("#integration-url-config-options-container");
+            $configContainer.empty();
+
+            for (const option of config) {
+                let $inputElement: JQuery;
+
+                if (option.validator === "check_bool") {
+                    $inputElement = $("<div>")
+                        .addClass("input-group")
+                        .attr("id", `integration-url-${option.key}-container`)
+                        .append(
+                            $("<label>")
+                                .addClass("checkbox")
+                                .append(
+                                    $("<input>")
+                                        .attr("type", "checkbox")
+                                        .attr("id", `integration-url-${option.key}`)
+                                        .addClass("integration-url-parameter")
+                                        .prop("checked", false),
+                                )
+                                .append($("<span>").addClass("rendered-checkbox")),
```

Comment:
Don't generate HTML with JS like this; use a template. Also, you should probably parse with `zod` to get the types right.